services/ui-src/src/utils/reports/entities.ts

```diff
@@ -61,6 +116,17 @@ export const getFormattedEntityData = (
       break;
 
     case ModalDrawerEntityTypes.QUALITY_MEASURES:
+      entityData = {
+        domain: getRadioValue(entity, "qualityMeasure_domain"),
+        name: entity?.qualityMeasure_name,
+        nqfNumber: entity?.qualityMeasure_nqfNumber,
+        reportingRateType: getReportingRateType(entity),
+        set: getRadioValue(entity, "qualityMeasure_set"),
+        reportingPeriod: getReportingPeriod(entity),
+        description: entity?.qualityMeasure_description,
+        numberOfPlans: reportFieldData?.plans.length,
```

Comment:
Note: Probably won't need this anymore.

Reply:
Also this is where the no-plans error is.
```suggestion
        numberOfPlans: reportFieldData?.plans?.length,
```